README.md

```diff
@@ -12,6 +12,29 @@ In addition to handling proposed answers, this action will automatically close a
 
 ![Action Settings in repo](images/repoTokenSettings.png)
 
+There is an alternative way to enable `write permissions` for the token instead of changing the Repository Settings. Required permissions could be added like this -
```

Comment:
I think we should have this be recommended way to add write permissions, since granting them in the workflow file grants the necessary permissions in a smaller scope. Least privilege and all 🙂 

Proposed reword: 

> A valid GitHub token is required to use this action. The GitHub token for your repository can be generated by GitHub during workflow execution when passed in as `${{ secrets.GITHUB_TOKEN }}`. The GitHub token must have `discussions: write` permissions, these permissions can be granted in the workflow file.

> Alternatively, write permissions can be granted to your repository's GitHub token for all workflows in your repository. Please check the permissions in your Repository `Settings -> Actions -> General` to grant write permissions to the token.

> image reference

Then, I don't think we need a duplicate workflow file example. We should just add the line to the existing example, and also fix the existing example by using `v1` instead of `main`